frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.ts

```diff
@@ -59,13 +70,25 @@ export class UserDetailsComponent extends BaseComponent implements OnInit {
                     sex: resp.sex,
                     language: resp.language,
                     languageLevel: resp.languageLevel,
+                    birthDate: resp.birthDate,
                 });
+
+                this.userService.getTagNames().pipe(this.untilThis)
+                    .subscribe(tags => this.selectedTags = tags);
             });
```

Comment:
Github actions

Reply:
Pls explain, not clear what you mean

Reply:
I just go through files
![image](https://user-images.githubusercontent.com/57347711/229287773-48011809-109a-447a-95cc-9b130647ca93.png)